settings/config.py

```diff
@@ -0,0 +1,22 @@
+import os
+import sys
+
+class Config(object):
+    DEBUG = False
+    TESTING = False
+    SECRET_KEY = "secret key"
+    UPLOAD_FOLDER = "./temp_uploads"
+    MAX_CONTENT_LENGTH = 16 * 1024 * 1024
+
+
+class ProductionConfig(Config):
+    pass
+
+
+class DevelopmentConfig(Config):
+    DEBUG = True
+    UPLOAD_FOLDER = "./uploads"
+
+
+class TestingConfig(Config):
+    TESTING = True
```

Comment:
What's this TESTING param used for? I realize it's in the Flask docs, but it doesn't seem like we are using it, so remove it.

Reply:
removed it